goose/00001_initial.sql

```diff
@@ -14,12 +14,12 @@ CREATE TABLE IF NOT EXISTS companies (
 
 CREATE TABLE IF NOT EXISTS persons (
   person_id         INT,
-  last_name         VARCHAR(255),
-  first_name        VARCHAR(255),
+  last_name         VARCHAR(100),
+  first_name        VARCHAR(100),
```

Comment:
this wouldn't work on mysql 5.6, where there's a 756 byte limit on indexes (utf8mb4, so 255 * 4 * 2)